README.md

```diff
@@ -1,89 +1,28 @@
 **DISCLAIMER:** This is not an official Google product.
 
 # OpenHTF
+The open-source hardware testing framework.
+
 [![Build Status](https://travis-ci.org/google/openhtf.svg?branch=master)](https://travis-ci.org/google/openhtf)
 [![Coverage Status](https://coveralls.io/repos/google/openhtf/badge.svg?branch=master&service=github)](https://coveralls.io/github/google/openhtf?branch=master)
 
 [![Issue Stats](http://issuestats.com/github/google/openhtf/badge/pr)](http://issuestats.com/github/google/openhtf)
 [![Issue Stats](http://issuestats.com/github/google/openhtf/badge/issue)](http://issuestats.com/github/google/openhtf)
 
-The open-source hardware testing framework.
+
+## Overview
```

Comment:
hm, i don't want to nit via comment because the RTT is too slow, but for PyPI launch we should spend some time and craft this first paragraph very intentionally, as it's likely the most visible thing to first-time users.  This is a good first pass at it, though.